pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -83,8 +109,10 @@ in stdenv.mkDerivation {
     "-Dtests=false"
     "-Dimportd=true"
     "-Dlz4=true"
+    "-Dhomed=false"
```

Comment:
Can't we either make this configurable with a package option or add e.g. a `systemd-experimental`-package? I guess that enabling those features in e.g. the local `systemd` might be helpful when testing those new features (we should note in the docs however that we don't support `homed` and `portabled` atm and whoever uses this should expect things to break).

Reply:
I'd prefer to set this to true, once it's verified to work and there's a test for it, but deliberately set it to false until them (as stated in the commit message).

I don't see much value in providing another systemd package for "experimental/untested features" - I'd expect people fiddling with this to be able to import systemd from their wip checkout.

Reply:
Fair enough :)